ci/builders/web_engine.json

```diff
@@ -21,7 +30,24 @@
         "tests": []
     },
     {
-        "archives": [],
+        "archives": [
+		{
+			"name": "linux_host_debug_unopt",
+			"include_paths": ["out/host_debug_unopt"],
+			"exclude_paths": [
+				"out/host_debug_unopt/obj",
+				"out/host_debug_unopt/exe.unstripped"
+			]
+		},
+		{
+			"name": "javascript_tests",
+			"include_paths": [
+				"flutter/lib/web_ui/build/test",
+				"flutter/web_sdk/web_engine_tester/lib/static"
```

Comment:
I just realized that we also need to include `lib/web_ui/.dart_tool/goldens` in the archive.